\section{Discussion}

Our goal here is to highlight the potential for GoM models to elucidate
structure in RNA-seq data from both single cell sequencing and bulk sequencing of pooled cells. 
We also provide tools to identify which genes
are most distinctively expressed in each cluster, to aid interpretation of results.
As our applications illustrate, these methods have the potential to highlight biological processes
underlying the cluster structure identified.

The GoM model has several advantages over distance-based hierarchical methods of clustering. At the most basic level model-based methods are often
more accurate than distance-based methods. Indeed, in our simple test on the GTEx data the model-based GoM approach
more accurately separated samples into ``known" clusters. However, there are also other subtler benefits of the GoM model. Because the GoM 
model does not assume a strict ``discrete cluster" structure, but 
rather allows that each sample has a proportion of membership in each cluster, it can provide insights into how well a 
particular dataset really fits a ``discrete cluster" model. For example, consider the results for the data from Jaitin \textit{et al} \cite{Jaitin2014} and Deng \textit{et al} \cite{Deng2014}:
in both cases most samples are assigned to multiple clusters, although the results are closer to ``discrete" for the latter than the former.
The GoM model is also better able to represent the situation where there is not really a single clustering of the samples,
but where samples may cluster differently at different genes. For example, in the GTEx data, the stomach samples share memberships in common with
both the pancreas (purple) and the adrenal gland (light green). This pattern can be seen in the Structure plot (Fig \ref{fig:fig1}) but would be hard to 
discern from a standard hierarchical clustering. 

%On the other hand, hierarchical clustering methods do have a natural advantage when capturing structure that is naturally hierarchical.
%Deconvolution techniques using marker genes are popularly used to learn about the the concentration of different cell types in a cell mixture and cell type signature expression profiles. Our technique is analogous to blind deconvolution approach which estimates the cell type proportions and cell type signatures jointly (see Schwartz \textit{et al} 2010 \cite{Schwartz2010} and Repsilber \textit{et al} 2010 \cite{Repsilber2010}), except that we operate under Poisson model framework to model the counts data. This fully unsupervised approach, however, seems to fail in identifying clusters determined completely by individual cell types. Proposed alternatives to blind deconvolution approach include two-pronged sequential updating of the cell type concentration and cell type proportion (see Lindsay \textit{et al} \cite{Lindsay2013}) and supervised or semi-supervised learning where it is assumed that some information of cell type signature expression profiles is known (see Shen-Orr \textit{et al} 2010 \cite{Shen-Orr2010},  Qiao \textit{et al} 2012 \cite{Qiao2012}). As part of our future works, we plan to implement similar modifications to our method so as to make the clusters more biologically interpretable. \\[1 pt]

%However, the Structure plot in  \textbf{Fig \ref{fig:fig2}} and the corresponding cluster annotation (\textbf{Supplementary Table 1}) seem to indicate that the clusters may be \textit{driven} by different cell types, which is encouraging.

%Since our method is model based, it provides an optimal $K$ for the model fit. However, one has to run the model on the data for a range of $K$'s and that is not always practical when running the model on large datasets as in RNA-seq reads data. 

GoM models also have close connections with dimension reduction techniques such as factor analysis, principal components analysis and non-negative matrix factorization. All of these methods can also be used for RNA-seq data, and may often be useful. See  \cite{Engelhardt2010} for discussion of 
relationships among these methods in the context of inferring population genetic structure. While not arguing that the GoM model
is uniformly superior to these other methods, we believe our examples illustrate the appeals of the approach. In particular, we would argue that for the GTEx data, 
the Structure plot (Fig \ref{fig:fig1}) combined with the cluster annotations (Table \ref{tab:tab1}) provide a more visually and biologically appealing summary of the data
than would a principal components analysis.

Fitting GoM models can be computationally-intensive for large data sets.
For the datasets we considered here the computation time ranged from $12$ minutes for the data from \cite{Deng2014} ($n=259; K=6$), through $33$ minutes for the 
data from \cite{Jaitin2014} ($n=1,041; K=7$) to 3,370 minutes for the GTEx data ($n=8,555; K=20$). 
%maybe add comparison with hierarchical model (346 mins on GTEx data)
Computation time can be reduced by fitting the model to only the most highly expressed genes, and we often use this strategy to get quick initial results for a dataset.
Because these methods are widely used for clustering very large document datasets there is considerable ongoing interest in 
computational speed-ups for very large datasets, with ``on-line" (sequential) approaches capable of dealing with millions of documents  \cite{Hoffman2010} that could be useful in the future for very large RNA-seq datasets. 

A thorny issue that arises when fitting these types of model is how to select the number of clusters, $K$. Like many software packages
for fitting these models, the {\tt maptpx} package implements a measure of model fit that provides one useful guide. However, it is
worth remembering that in practice there is unlikely to be a ``true" value of $K$, and results from different 
values of $K$ may complement one another rather than merely competing with one another. For example, seeing how the fitted model evolves
as $K$ increases is one way to capture some notion of hierarchy in the clusters identified \cite{Rosenberg2002}. More generally it is often
fruitful to analyse data in multiple ways using the same tool: for example our GTEx analyses illustrate how 
analysis of subsets of the data (in this case the brain samples) can complement analyses of the entire data.

The version of the GoM model fitted here is relatively simple, and could certainly be embellished. For example,
the model allows the expression of each gene in each cluster to be a free parameter, whereas we might expect expression of most
genes to be ``similar" across clusters. This is analogous to the idea in population genetics applications that allele frequencies in different populations may
be similar to one another \cite{Falush2003}, or in document clustering applications that most words may not differ appreciably in frequency in different topics. 
In population genetics applications incorporating this idea into the model, by using a correlated prior distribution on these frequencies, 
can help improve identification of subtle structure \cite{Falush2003} and we would expect the same to happen here for RNA-seq data.



%The user is only required to input the matrix of counts obtained from RNA-seq  reads mapping to genes, along with the sample metadata and a set of $K$'s or the number of clusters he wants to fit, and the output would include the estimates of the model parameters, along with the Structure plot visualizations ordered by sample metadata and the set of most informative genes across the different clusters. 
%


%
%that takes as input the read counts matrix over the samples and genes and number of clusters to fit ($K$), and gives as output the admixture proportions matrix for all the samples and the relative expression profiles of the genes in each of the $K$ clusters. It also provides us with a model log-likelihood that can be used to choose the optimal $K$ to fit. However for genetic data, it is more recommended to observe the clustering patterns over a range of values of $K$ to observe how the patterns change as we increase $K$. The clustering method is pretty fast as it uses  EM algorithm along with quasi-Newton updates to speed up the iterations. The clustering proportions obtained as output can be viewed using a Structure plot or the t-SNE plot that give a much better visual representation of the clustering patterns than heatmap or PCA. As per model specifications, ideally the clusters should be driven by the cell types and we already have seen some evidence in support of that in \textbf{Fig \ref{fig:fig2}} when the model was applied on brain samples. Besides the cluster proportions, the model also provides the user with the relative expression profile of all genes in each of these clusters, from which it is easy to figure out which genes have significantly high expression in one or more clusters compared to the other clusters or in other words, are informative in driving the clusters. The user can select these cluster driving genes and annotate them to get a better understanding of the biological significance of the clusters.  Even purely as a clustering technique, our method  outperforms hierarchical method in separating out the samples belonging to distinct classes (in case of the GTEx data, the different tissues). So, overall we feel our model has a number of advantages over the standard methods of clustering used in RNA-seq or scRNA-seq literature, like hierarchical clustering, in terms of cluster quality, biological validation, visualization and interpretation.  The clustering along with the Structure plot representation based on the sample metadata is implemented in package \textbf{CountClust} available on Github (\href{https://github.com/kkdey/CountClust}{https://github.com/kkdey/CountClust}) which is a wrapper package of \textbf{maptpx} due to Matt Taddy \cite{Taddy2012}.  \\[3 pt]
%
%
%\textbf{Future works}
%
%\begin{itemize}
%\item It will be worthwhile to see if instead of finding out cluster driving genes, we can find out cluster driving gene pathways. which would have significance from a biomedical standpoint. 
%
%\item Since many of the genes are not informative for the clustering, we may try to impose a variable selection preprocessing or incorporate that in our model suitably so that it will extract out only the genes that are informative about the clusters and will also speed up the  model fitting. 
%
%\item The admixture proportions may be useful for determining the mixing weights for the prior covariance matrices in the eQtlbma. 
%
%\item We may have important metadata on the samples (for instance the individual from whom the sample came from) or on the genes (for instance the gene length, GO or KEGG annotations, GC content etc) which we have not incorporated in our clustering model so far. 
%





%\subsection{Normalization issue}
%
%A common practice in RNA-seq literature is to normalize the counts data by the library size before applying any differential analysis or clustering methods to it. Depending on sequencing machine/ lane use or change in sequencing depth, it may happen that some samples have very high counts  across  most genes, while some other samples may have very low counts of reads across all genes. This can lead to severe bias in statistical analysis if not accounted for. A way to counter this, given the raw counts, is to use the CPM (counts per million) normalized data \cite{Robinson2010} \cite{Law2014}. We define the CPM normalized data $X_{ng}$ as 
%
%$$  X_{ng} = \frac{c_{ng}}{ \left [ \frac{L_{n}}{10^6} \right ] } $$
%
%Here  $L_n$ is the library size or the total sum of the counts of all reads for the sample $n$. Though we apply our clustering algorithm on raw counts, we claim that CPM-normalization is intrinsic to the method. In mathematical terms, we are trying to model 
%
%$$ p_{ng} = \sum_{k=1}^{K} q_{nk}\theta_{kg}   \hspace{1 cm}  \sum_{k=1}^{K} q_{nk}=1 \hspace{1 cm} \sum_{g=1}^{G} \theta_{kg}=1 $$
%
%A very naive estimate of $p_{ng}$ would be 
%
%$$ \hat{p}_{ng} = \frac{c_{ng}}{L_{n}} = \frac{X_{ng}}{10^6}  $$
%
%Therefore clustering with respect to $p_{ng}$  ideally takes into account the variation at the $X_{ng}$ level or normalized counts level instead of at $c_{ng}$ level.  From an information theoretic point of view, it seems that all information required for the clustering is contained in $X_{ng}$'s. 
%
%\subsection{Admixture model vs Hierarchical model}
%
%Admixture model is a model based soft clustering method and hierarchical model is a non-model based tree clustering method and it is difficult to find a common measure that can effectively compare these two clustering schemes. We use real data to compare between the two methods. We drew 50 tissue samples from the pool of Muscle-Skeletal and Heart- Left Ventricle samples in GTEx Version 4 gene counts data with thinning parameter ($p_{thin}$) equal to $0.0001$  and compared the heatmap of the admixture proportions $q$ computed from the Admixture model with $k=2$ clusters, with the hierarchical clustering heatmap on the counts data. It seems that admixture reduces the noise in the high dimensional data and does a better job at segregating the tissue samples corresponding to Muscle-Skeletal and Heart Left-Ventricle (Fig ??). 
%
%\subsection{Batch effects}
%
%One of the important factors that may impact the clustering or differential analysis in tissue level RNA-seq or single cell RNA-seq analysis are technical or batch effects. These batch effects may stem from data coming from different laboratories or even for a single laboratory experiment, there may be effects due to the sequencing lane used , or the plate chosen for the experiment or the amplification process adopted. There has been a growing concern among biostatisticians today regarding how to deal with batch effects \cite{Leek2010} \cite{Hicks2015}. For our clustering method as well, batch effects are an important concern. We present a case study where we used the Admixture model on a massively paralllel single cell RNA-seq data obtained from mouse spleen by Jaitin \textit{et al} 2015,   with the aim to replicate the clustering results reported in the paper \cite{Jaitin2015}. However, following the experimental metadata provided by the authors, we observed that the data coming from the same sequencing or amplification batch seemed to have very similar patterns and it seemed that the biological effects may be confounded with the batch effects (Fig ??). To top that there was also a complete confounding between the amplification batch and then sequencing batch. So, one needs to be cautious about jumping to conclusions about clustering patterns just by looking at the Structure plot. We strongly suggest careful investigation of the experimental details to determine if there are any batch effects and also gene annotations to observe if the clustering method is indeed driven by genes that have biological functions relevant to the clustering patterns.
%
%\subsection{Thinning}
%
%Since we have explored both tissue level and single cell level RNA-seq data for our clustering, we were curious to what extent the clustering patterns in GTEX V4 tissue level data would have been retained if we had observed single cell level data instead of tissue samples data. This is why we used the thinning of the GTEX dataset, by using various degrees of thinning. An objective choice of the thinning parameter for us would be $p_{thin}=0.001$, which was the order of the ratio of total number of reads for the Jaitin \textit{et al} single cell data and the GTEX V4 tissue level data \cite{GTEX2013} \cite{Jaitin2015}. It is pretty obvious that the more we thin the data, the more we will lose the clustering patterns and for $p=0$, we shall have only zero counts for all samples. We tried thinning parameters $p_{thin}=0.1,0.01,0.001$ and found that the patterns seemed more or less similar although a few of the patterns indeed disappeared as we thinned the data more and more. Besides the single cell analogy, there is a computational reason one might be motivated to do thinning. Very large counts with intermittent zeros in tissue level data may lead to large overdispersion and a bad fit for our model, while smaller counts as in the thinned data checks the dispersion factor from blowing up. Secondly, running the admixture model on the thinned data is computationally much faster. However, one must be cautious regarding the extent of thinning and it is always a good idea to check the patterns for multiple values of the thinning parameter $p_{thin}$ and note how robust the clustering results are to the change in thinning parameter. 
% 
%

%\subsection{Performance analysis of the Admixture model}
%
%We carried out a few simulation studies to analyze the performance of the Admixture model under different scenarios. The main focus was to figure out how sensitive the method is to the admixture proportions or the relative gene expression differences. For instance, if we have data coming from 2 clusters, then under how well does our model do under different choices of the admixture proportions and the gene expression. We first consider a scenario where we have 1000 samples and 500 genes and the admixture proportion for sample $n$ is of the form $(\frac{n}{1000}, \frac{1000-n}{1000})$ and the allele frequency vectors $\theta_1$ and $\theta_2$  for the two clusters are given by 
%
%$$ \theta_1 = (0.01, 0.05, \frac{0.94}{498}, \frac{0.94}{498}, \cdots, \frac{0.94}{498}) $$
%$$ \theta_2 = (0.05, 0.01, \frac{0.94}{498}, \frac{0.94}{498}, \cdots, \frac{0.94}{498}) $$
%
%The true admixture graph and the estimated admixture graph on the simulated counts table for $1000$ samples and $500$ genes is provided in Fig ?? (top panel). The top  two significantly enriched genes for the clustering (as per Subsection 3.3) were found to be genes $1$ and $2$, which is clearly the case. 
%
%Now we consider a second scenario with the same set up as before  but now the admixture proportion for $n$ th  sample is of the form $(0.4 +0.2 \times \frac{n}{1000}, 0.6 -0.2\times \frac{n}{1000})$. This means that the variation in admixture proportions is less compared to the previous set up. The true admixture graph and the estimated admixture graph on the simulated counts table under this set up is presented in Fig ?? (bottom panel). The top significantly enriched genes for the clustering were found to be $483$ and $224$ which are way off.  This shows that keeping the gene expression the same, admixture model is more successful in detecting clusters with large differences in admixture proportions between them. 
%
%Next we present a phase diagram analysis to show that the sensitivity of the admixture model depends both on how close the admixture proportions of the two subgroups or clusters are as well as how close the relative gene expression patterns for the two clusters are. For this analysis, we again assume that we have $K=2$ clusters, we chose the number of samples to be $200$ and we vary over the number of genes to be $50,100$ and $200$. We assume two phase parameters, $\alpha$ and $\gamma$. $\alpha$ is the phase parameter for the admixture proportions and we assume that for sample $n$, the admixture proportion is of the form $(\alpha, 1-\alpha)$. On the other hand, for $G$ genes, we assume that the phase parameter $\gamma < \frac{2}{G}$ and the relative gene expression $\theta_1$ and $\theta_2$ are of the form 
%
%$$ \theta_1 =  \left ( \gamma, \frac{2}{G} - \gamma, \frac{1}{G}, \frac{1}{G}, \cdots, \frac{1}{G} \right ) $$
%$$ \theta_2 =  \left ( \frac{2}{G} - \gamma, \gamma, \frac{1}{G}, \frac{1}{G}, \cdots, \frac{1}{G} \right ) $$
%
%We varied over $\alpha$ and $\gamma$, for each choice generated a counts table with $200$ samples and $G$ genes and then carried out admixture. We then observed whether the estimated admixture proportions match with the true proportions and whether we are detecting the significantly enriched genes for clustering, in this case genes $1$ and $2$. If $\Omega$ is the $ N \times K$  true topic proportion matrix and $\hat{\Omega}$ be the estimated topic proportion matrix. We say \textit{omega match} if 
%
%$$ min \left ( || \hat{\Omega} - \Omega ||_2, ||1 - \hat{\Omega} - \Omega ||_2  \right )  < 0.05 $$
%
%We say \textit{theta match} if the top two genes detected by the clustering method to be driving the clusters (as per Subsection 3.3) are genes 1 and 2. We present the phase diagram to see for which values of $\alpha$ and $\gamma$, there is \textit{omega match} or \textit{theta match} or both (Fig ??).
%
%
%Finally, it must also be emphasized that if there are more than two clusters in the data and we fit just $2$ clusters, then the clusters obtained may not be driven by any of the real clusters but by a mix of the clusters present. We performed a simulation scenario where we have samples coming from a mix of $5$ clusters with varying admixture proportions (true admixture proportion design shown in Fig ?? (a) ). The relative gene expression profiles  of $500$ genes considered for the $5$ clusters were assumed to be 
%
%$$ \theta_1 = (0.01,0.02,\frac{0.97}{498}, \frac{0.97}{498}, \cdots, \frac{0.97}{498}) $$
%$$ \theta_2 = (\frac{0.98}{498}, \frac{0.98}{498}, \cdots, \frac{0.98}{498}, 0.01,0.01) $$
%$$ \theta_3 = (\frac{0.97}{498}, \frac{0.97}{498}, \cdots, \frac{0.97}{498}, 0.01,0.02) $$
%$$ \theta_4 = (0.01, 0.01, 0.1,0.2, \frac{0.68}{498}, \frac{0.68}{498}, \cdots, \frac{0.68}{498}) $$
%$$ \theta_5 =(0.1,0.1,0.1,0.1,0.1, \frac{0.5}{495}, \frac{0.5}{495}, \cdots, \frac{0.5}{495}) $$
%
%
%We drew $1000$ samples from the above set up of admixture proportions and relative gene expression of the clusters for $500$ genes and then applied the admixture model to the counts data. The true and the estimated Structure plot are presented in Fig ??. 
%
%
% 

 













