\section{Methods and Materials}

% We also remove spike-in control genes,  as the latter may create bias due to their typically having high number of reads mapped to them \cite{Jiang2011}. 

%RNA-seq experiments provide us with a set of FASTQ files that contain the nucleotide sequence of each read and a quality score at each position, which can be mapped to  reference genome or exome or transcriptome. The output of this mapping is usually saved in a SAM/BAM file using SAMtools  \cite{Li2009}, a task primarily accomplished by \textit {htseq-counts}  by Sanders et al  2014 \cite{Sanders2014} or \textit{featureCounts}  [ R package \textbf{Rsubread} ] by Liao et al 2013 \cite{Liao2013}.  RNA-seq raw counts are the basis of all statistical workflows, be it exploration or differential expression analysis [\textbf{edgeR} \cite{Robinson2010}, \textbf{limma} \cite{Ritchie2015} ]. There is a growing trend to make the analysis ready raw counts tables openly accessible for statistical analysis. ReCount is a online site that hosts RNA-seq gene counts datasets from 18 different studies \cite{Frazee2011} along with relevant metadata. Such gene counts datasets are the inputs for our clustering algorithm. \\[2 pt]
%

% In the preprocessing step before applying our method, we remove the genes with 0 or same count of matched reads across all samples (non-informative genes), any sample or gene  with NA values of reads and  ERCC spike-in controls,  as the latter may create bias due to their typical very high expression (number of reads mapped to them).  For illustration, we applied our method GTEx Version 6 tissue level gene counts data \cite{GTEX2013} and on a couple of single cell data due to Zeisel \textit{et al} \cite{Zeisel2015} and Jaitin \textit{et al} \cite{Jaitin2014}. 
 
% If $C_{ng}$ is the gene count for $g$ th gene in tissue sample $n$, then we define the thinned counts as 
%
%$$ c_{ng}  \sim Bin(C_{ng}, p_{thin} )  $$
%
%where $p_{thin}$ is the thinning probability. W chose $p_{thin}$ to be of the order of the ratio of the total number of reads mapped to a single cell experiment (in this case Zeisel et al (2015) data for instance) and the total number of reads in the GTEx dataset, which turned out to be approximately 0.0001. To check for robustness of our clustering algorithm, we varied $p_{thin}$ to be $0.01, 0.001, 0.0001$ (see Fig ).  
%

\subsection{Model Fitting}

We use the {\tt maptpx} R package \cite{Taddy2012} to fit the GoM model (\ref{eqn:mult},\ref{eqn:gom}), which is also known as ``Latent Dirichlet Allocation" (LDA). 
The {\tt maptpx} package fits this model using an EM algorithm to perform Maximum a posteriori (MAP)  estimation of the parameters $q$ and $\theta$. See \cite{Taddy2012} for details.

%This model has $N \times (K-1) + K \times (G-1)$ parameters, which is much smaller than the $N \times G$  data values of counts. Usually for RNA-seq samples $N$ varies in the region of $100$s to $1000$s  and $G$ ranges from $10,000$ to $50,000$ (depending on the underlying species and the types of genes tokenized) and $K << \{N,G \}$. 

% It assumes the priors
%
%$$ q_{n*} \sim Dir ( \frac{1}{K}, \frac{1}{K}, \cdots, \frac{1}{K} ) $$
%$$ \theta_{k*} \sim Dir(\frac{1}{KG}, \frac{1}{KG}, \cdots, \frac{1}{KG} ) $$
%
%For better estimation stability, the usual parameters of the model are converted to natural exponential family parameters to which one can apply the EM algorithm ). The value of the Bayes factor for the model with $K$ clusters compared to the model with 1 cluster, is recorded for each $K$, and the optimal $K$ is chosen by running the clustering method for different choices of $K$ and then choosing the one with maximum Bayes factor. The two main outputs from this method are the $Q_{N \times K}$ topic proportion matrix  and $F_{K \times G}$ relative gene expression for each cluster.

\subsection{Visualizing Results}

%We visualize results using a ``Structure plot" \cite{Rosenberg2002}, 
%which is named for its widespread use in visualizing the
%results of the ``structure" software \cite{Pritchard2000} in population genetics.
%The Structure plot represents each GoM vector $q_{n\cdot}$
%as  a vertical stacked barchart, with bars of different colors representing membership proportion in each cluster (e.g.~Figure \ref{fig:fig1}). If the colored patterns of two bars are similar, then the two samples have similar membership proportions.  The Structure plot is particularly helpful when external information is available on each sample that can be used to order or group the samples in an informative way.

In addition to the Structure plot, we have also found it useful to visualize results using t-distributed Stochastic Neighbor Embedding (t-SNE), which is a method for visualizing high dimensional datasets by placing them in a two dimensional space, attempting to preserve the relative distance between nearby samples \cite{Maaten2014,Maaten2008}. Compared with the Structure plot our t-SNE plots contain less information, but can better emphasise clustering of samples that have similar membership proportions in many clusters. Specifically, t-SNE tends to place samples with similar membership proportions together in the two-dimensional plot, forming visual ``clusters" that can be identified by eye (e.g. \url{http://stephenslab.github.io/count-clustering/project/src/tissues_tSNE_2.html}). This may be particularly helpful in settings where no external information is available to aid in making an informative Structure plot. 


\subsection{Cluster annotation}

To help biologically interpret the clusters, we developed a method to identify which genes are most distinctively differentially expressed in each cluster. 
(This is analogous to identifying ``ancestry informative markers" in population genetics applications \cite{Rosenberg2005}.)
 Specifically, for each cluster $k$ we measure the distinctiveness of gene $g$ with respect to any other cluster $l$ using
\begin{equation}
\KL^{g} [k,l] : = \theta_{kg} \; log \frac{\theta_{kg}}{\theta_{lg}} + \theta_{lg} - \theta_{kg},
\end{equation}
which is the Kullback--Leibler divergence of the Poisson distribution with parameter $\theta_{kg}$ to the Poisson distribution with parameter $\theta_{lg}$. 
For each cluster $k$, we then define the distinctiveness of gene $g$ as 
\begin{equation}
D^{g}[k] = \underset{l \neq k}{\min} \; \KL^{g} [k, l].
\end{equation}
The higher $D^g[k]$, the larger the role of gene $g$ in distinguishing cluster $k$ from all other clusters. 
Thus, for each cluster $k$ we identify the genes with highest $D^{g}[k]$ as the genes driving the cluster $k$. 
We annotate the biological functions of these individual genes using the {\tt mygene} R Bioconductor package  \cite{Thompson2014}. 


For each cluster $k$, we filter out a number of genes (top $100$ for the Deng \textit{et al} data \cite{Deng2014} and GTEx V6 data \cite{GTEX2013}) with highest $D^{g}[k]$ value and perform a gene set over-representation analysis of these genes against all the other genes in the data representing the background. To do this, we used ConsensusPathDB database (\url{http://cpdb.molgen.mpg.de/})  \cite{Kamburov2013} \cite{Pentchev2010}. See Table~\ref{tab:tab1}-\ref{tab:tab2} and Table~\ref{tab:tab3} for the top significant gene ontologies driving each cluster in the GTEx V6 data and the Deng \textit{et al} data respectively. 

\subsection{Comparison with hierarchical clustering}

We compared the GoM model with a distance-based hierarchical clustering algorithm by applying both methods to samples from 
pairs of tissues from the GTEx project, and assessed their accuracy in
separating samples according to tissue.  For each pair of tissues  we randomly selected $50$ samples from the pool of all samples coming from these tissues. 
For the hierarchical clustering approach we cut the dendogram at $K=2$, and checked whether or not this cut partitions the samples into the two tissue groups. 
(We applied hierarchical clustering using Euclidean distance, with both complete and average linkage; results were similar and so we showed results only for complete linkage.) 

For the GoM model we analysed the data with $K=2$, and sorted the samples by their membership in cluster 1. We then partitioned the samples at the point of the steepest fall in this membership, and again we checked whether this cut partitions the samples into the two tissue groups.


Fig~\ref{fig:fig2} shows, for each pair of tissues, whether each method successfully partitioned the samples into the two tissue groups.
%For instance, for GTEx tissue sample data, if the clusters are indeed driven by cell types, then the top genes for these clusters will probably be associated with proteins related to  functions for that particular cell type.
%

%we fix each gene and then look at the KL divergence matrix of one cluster/subgroup $k$ relative to other cluster/subgroup $k^{'}$, which we call $KL^{g}_{K \times K}$. This matrix is symmetric and has all diagonal elements $0$ as the divergence of a cluster with respect to itself is $0$. 

\subsection{Thinning}

We used ``thinning" to simulate lower-coverage data from the original higher-coverage data.. 
Specifically, if $c_{ng}$ is the counts of number of reads mapping to gene $g$ for sample $n$ for the original data, we simulated 
thinned counts $t_{ng}$ using 
\begin{equation}
t_{ng} \sim Bin (c_{ng}, p_{thin})
\end{equation}
where $p_{thin}$ is a specified thinning parameter.


%Then we perform gene annotations for the top genes in each subgroup 
\subsection{Code Availability}

Our methods are implemented in an R package {\tt CountClust}, available as part of the Bioconductor project at \url{https://www.bioconductor.org/packages/3.3/bioc/html/CountClust.html}. The development version of the package is also available at \url{https://github.com/kkdey/CountClust}.

Code for reproducing results reported here is available at \url{http://stephenslab.github.io/count-clustering/}.


\textbf{\large{Acknowledgements}}

\footnotesize{The Genotype-Tissue Expression (GTEx) Project was supported by the Common Fund of the Office of the Director of the National Institutes of Health. Additional funds were provided by the NCI, NHGRI, NHLBI, NIDA, NIMH, and NINDS. Donors were enrolled at Biospecimen Source Sites funded by NCI $\ $ SAIC-Frederick, Inc. (SAIC-F) subcontracts to the National Disease Research Interchange (10XS170), Roswell Park Cancer Institute (10XS171), and Science Care, Inc. (X10S172). The Laboratory, Data Analysis, and Coordinating Center (LDACC) was funded through a contract (HHSN268201000029C) to The Broad Institute, Inc. Biorepository operations were funded through an SAIC-F subcontract to Van Andel Institute (10ST1035). Additional data repository and project management were provided by SAIC-F (HHSN261200800001E). The Brain Bank was supported by a supplements to University of Miami grants DA006227 $\&$ DA033684 and to contract N01MH000028. Statistical Methods development grants were made to the University of Geneva (MH090941 $\&$ MH101814), the University of Chicago (MH090951, MH090937, MH101820, MH101825), the University of North Carolina - Chapel Hill (MH090936 $\&$ MH101819), Harvard University (MH090948), Stanford University (MH101782), Washington University St Louis (MH101810), and the University of Pennsylvania (MH101822). The data used for the analyses described in this manuscript were obtained from: the GTEx Portal on 10/19/2015 and  dbGaP accession number phs000424.v6.p1.

The paper is supported by the grant U01CA198933 from the NIH BD2K program.

We thank Matt Taddy, Amos Tanay and Effi Kenigsberg for helpful discussions. We thank Po-Yuan Tung, John Blischak and Jonathan Pritchard for helpful comments on the draft manuscript.}

\textbf{\large{Disclosure Declaration}}

\footnotesize{The authors have no conflict of interest. }















 









