\clearpage

\begin{thebibliography}{9}

%\bibitem{Sanders2014} 
%S Anders, T P Pyl, W Huber.
%\textit{HTSeq :  A Python framework to work with high-throughput sequencing data}. 
%Bioinformatics, 2014, in print; online at doi:10.1093/bioinformatics/btu638
%
%\bibitem{Li2009} 
%Li H.*, Handsaker B.*, Wysoker A., Fennell T., Ruan J., Homer N., Marth G., Abecasis G., Durbin R. and 1000 Genome Project Data Processing Subgroup.
%\textit{The Sequence alignment/map (SAM) format and SAMtools}. 
%Bioinformatics, 2009, 25, 2078-9. [PMID: 19505943]
%
%\bibitem{Liao2013}
%Liao Y, Smyth GK and Shi W. 
%\textit{The Subread aligner: fast, accurate and scalable read mapping by seed-and-vote}.
%Nucleic Acids Research, 2013, 41, pp. e108.

%\bibitem{Robinson2010}
%Robinson MD, McCarthy DJ and Smyth GK. 
%\textit{edgeR: a Bioconductor package for differential expression analysis of digital gene expression data}.
%Bioinformatics, 2010, 26, pp. -1.

%\bibitem{Ritchie2015}
%Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W and Smyth GK. 
%\textit{limma powers differential expression analyses for RNA-sequencing and microarray studies}.
%Nucleic Acids Research, 2015, 43(7), pp. e47.

\bibitem{Hernandez1997}
Hernandez MC , Andres-Barquin PJ , Martinez S , Bulfone A , Rubenstein JL , Israel MA. 1997.
ENC-1: a novel mammalian kelch-related gene specifically expressed in the nervous system encodes an actin-binding protein.
\textit{J Neurosci.},17(9): 3038-51.

\bibitem{Baba1997}
Baba H, Nakahira K, Morita N, Tanaka F, Akita H, Ikenaka K.
GFAP gene expression during development of astrocyte.
\textit{Dev Neurosci.}, 19(1):49-57.

\bibitem{Eisen1998}
Eisen MB, Spellman PT, Brown PO and Botstein D. 1998.
Cluster analysis and display of genome-wide expression patterns.
\textit{PNAS}, 95(25): 14863-14868

\bibitem{Golub1999}
Golub TR, Slonim DK, Tamayo P, Huard C, Gaasenbeek M, Mesirov JP, Coller H, Loh ML, Downing JR, Caligiuri MA, Bloomfield CD, Lander ES. 1999.
Molecular classification of cancer: class discovery and class prediction by gene expression monitoring.
\textit{Science}, 286(5439): 531-7

\bibitem{Alizadeh2000}
Alizadeh AA1, Eisen MB, Davis RE, Ma C, Lossos IS, Rosenwald A, Boldrick JC \textit{et al}. 2000.
Distinct types of diffuse large B-cell lymphoma identified by gene expression profiling.
\textit{Nature}, 403(6769):  503-11

\bibitem{D'haeseleer2005}
D'haeseleer P. 2005.
How does gene expression clustering work?
\textit{Nat Biotechnol}, 23(12):1499-501

\bibitem{Jiang2004}
Jiang D, Tang C, Zhang A.
Cluster Analysis for Gene Expression Data: A Survey. 
\textit{Microsoft Research}, http://research.microsoft.com/en-us/people/djiang/tkde04.pdf.

\bibitem{Erosheva2006}
Erosheva EA. 2006.
Latent class representation of the grade of membership model.
Seattle: University of Washington.

\bibitem{Tang2009}
Tang F,  Barbacioru C,  Wang Y,  Nordman E,  Lee C  et al. 2009.
mRNA-Seq whole-transcriptome analysis of a single cell.
\textit{Nature Methods}, 6, 377 - 382.

\bibitem{Trapnell2015}
Trapnell C. 2015.
Defining cell types and states with single-cell genomics.
\textit{Genome Res.},  25, 1491-1498.

%\bibitem{Jiang2011}
%Jiang L, Schlesinger F, Davis CA, Zhang Y, Li R, Salit M, Gingeras TR and Oliver B. 
%\textit{Synthetic spike-in standards for RNA-seq experiments}.
%Genome Res, 2011:21, 1543-1551

%\bibitem{Frazee2011}
%Frazee AC, Langmead B, Leek JT. 
%\textit{ReCount: a multi-experiment resource of analysis-ready RNA-seq gene count datasets}. 
%BMC Bioinformatics, 2011, 12:449.

%\bibitem{Zeisel2015}
%Amit Zeisel, Ana B. Muñoz-Manchado, Simone Codeluppi, Peter Lönnerberg, Gioele La Manno, Anna Juréus, Sueli Marques, Hermany Munguba, Liqun He, Christer Betsholtz, Charlotte Rolny, Gonçalo Castelo-Branco, Jens Hjerling-Leffler, and Sten Linnarsson.
%\textit{Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq}.
%Science 6 March 2015: 347 (6226), 1138-1142.

\bibitem{GTEX2013}
The GTEx Consortium. 2013.
The Genotype-Tissue Expression (GTEx) project. 
\textit{Nature genetics}. 45(6): 580-585. doi:10.1038/ng.2653.

\bibitem{Oshlack2010}
Oshlack A, Robinsom MD, Young MD. 2010.
From RNA-seq reads to differential expression results.
\textit{Genome Biology}. 11:220, DOI: 10.1186/gb-2010-11-12-220

\bibitem{Taddy2012}
Matt Taddy. 2012.
On Estimation and Selection for Topic Models. 
\textit{AISTATS 2012, JMLR W\&CP 22}.
(maptpx R package).

\bibitem{Pritchard2000}
Pritchard, Jonathan K., Matthew Stephens, and Peter Donnelly. 2000.
Inference of population structure using multilocus genotype data. 
\textit{Genetics}. 155.2,  945-959.

\bibitem{Rosenberg2002}
Rosenberg NA, Pritchard JK,  Weber JL, Cann HM,  Kidd KK,  Zhivotovsky LA,  Feldman MW. 2002.
The genetic structure of human populations. 
\textit{Science}. 298,  2381-2385. 

\bibitem{Rosenberg2005}
Rosenberg NA. 2005.
Algorithms for selecting informative marker panels for population assignment.
\textit{J Comput Biol}. 12(9), 1183-201.

\bibitem{Raj2014}
Raj A,  Stephens M,  Pritchard JK.  2014.
fastSTRUCTURE: Variational Inference of Population Structure in Large SNP Data Sets.
\textit{Genetics}. 197,  573-589.

\bibitem{Falush2003}
Falush D,  Stephens M,  Pritchard JK. 2003.
Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.
\textit{Genetics}. 164(4), 1567-87.

\bibitem{Engelhardt2010}
Engelhardt BE,  Stephens M. 2010.
Analysis of Population Structure: A Unifying Framework and Novel Methods Based on Sparse Factor Analysis.
\textit{PLOS Genetics}. DOI: 10.1371/journal.pgen.1001117.

\bibitem{Maaten2008}
van der Maaten LJP and  Hinton GE. 2008.
Visualizing High-Dimensional Data Using t-SNE. 
\textit{J. Mach. Learn. Res.}.  2579-2605.

\bibitem{Maaten2014}
L.J.P. van der Maaten. 2014.
Accelerating t-SNE using Tree-Based Algorithms. 
\textit{J. Mach. Learn. Res.}.  3221-3245.

%\bibitem{Thompson2014}
%Mark A, Thompson R and Wu C. 
%\textit{mygene: Access MyGene.Info services}. 
%2014. R package version 1.2.3.

%\bibitem{Law2014}
%Law CW, Chen Y, Shi W, Smyth GK. 
%\textit{voom: precision weights unlock linear model analysis tools for RNA-seq read counts}. 
%Genome Biology. 2014;15(2):R29. 

\bibitem{Jaitin2014}
Jaitin DA,  Kenigsberg E et al. 2014.
Massively Parallel Single-Cell RNA-Seq for Marker-Free Decomposition of Tissues into Cell Types.
\textit{Science}. 343 (6172) 776-779.

\bibitem{Deng2014}
Deng Q,  Ramskold D,  Reinius B,  Sandberg R. 2014.
Single-Cell RNA-Seq Reveals Dynamic, Random Monoallelic Gene Expression in Mammalian Cells.
\textit{Science}.  343 (6167) 193-196.


\bibitem{Leek2007}
Leek JT,  Storey JD. 2007.
Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis
\textit{PLoS Genet}. 3(9): e161. doi:10.1371/journal.pgen.0030161

\bibitem{Stegle2012}
Stegle O,  Parts L ,  Piipari M,  Winn J,  Durbin R. 2012.
Using probabilistic estimation of expression residuals (PEER) to obtain increased power and interpretability of gene expression analyses.
\textit{Nat Protoc.}. 7(3):500-7. doi: 10.1038/nprot.2011.457.


\bibitem{Leek2010}
Leek JT,  Scharpf RB,  Bravo HC,  Simcha D,  Langmead B,  Johnson WE,  Geman D,  Baggerly K,  Irizarry RA. 2010.
Tackling the widespread and critical impact of batch effects in high-throughput data.
\textit{Nature Reviews Genetics}. 11, 733-739.

\bibitem{Hicks2015}
Hicks SC, Teng M, Irizarry RA. 2015.
On the widespread and critical impact of systematic bias and batch effects in single-cell RNA-Seq data.
\textit{BiorXiv}. http://biorxiv.org/content/early/2015/09/04/025528

%\bibitem{Houzel2005}
%Herculano-Houzel S and Lent R.
%\textit{Isotropic fractionator: a simple, rapid method for the quantification of total cell and neuron numbers in the brain}.
%J Neurosci. 2005 Mar 9;25(10), 2518-21.

\bibitem{Thompson2014}
Mark A, Thompson R and Wu C.  2014.
mygene: Access MyGene.Info services. 
\textit{R package version 1.2.3.}. 

\bibitem{Gentleman2003}
Gentleman, R., Bates, D., Bolstad, B \textit{et al}.
 Bioconductor: a software development project. 2003.
 \textit{Technical Report, Department of Biostatistics, Harvard
School of Public Health, Boston}. \url{https://bioconductor.org/}

%
%\bibitem{Grün2015}
%Grün D, Lyubimova A, Kester L, Wiebrands K, Basak O,  Sasaki N,  Clevers H, van Oudenaarden A.
%\textit{Single-cell messenger RNA sequencing reveals rare intestinal cell types}.
%Nature. 2015 Sep 10;525(7568), 251-5.

%\bibitem{Buettner2015}
%Buettner F, Natarajan KN, Casale FP, Proserpio V, Scialdone A, Theis FJ, Teichmann SA, Marioni JC and Stegle O.
%\textit{Computational analysis of cell-to-cell heterogeneity in single-cell RNA-sequencing data reveals hidden subpopulations of cells}
%Nature Biotechnology 2015, 33, 155?160, doi:10.1038/nbt.3102

%\bibitem{Palmer2005}
%Palmer C, Diehn M, Alizadeh AA and Brown PO.
%\textit{Cell-type specific gene expression profiles of leukocytes in human peripheral blood}.
%BMC Genomics 2006, 7:115.

\bibitem{Flutre2013}
Flutre T,  Wen X,  Pritchard J and Stephens M. 2013.
A Statistical Framework for Joint eQTL Analysis in Multiple Tissues.
\textit{PLoS Genet}. 9(5): e1003486. doi:10.1371/journal.pgen.1003486

\bibitem{Shiraishi2015}
Shiraishi Y, Tremmel G, Miyano S and Stephens M. 2015.
 A simple model-based approach to inferring and visualizing cancer mutation signatures.
\textit{ PLoS Genet}. 11(12): e1005657


\bibitem{Blei2003}
Blei DM,  Ng AY, Jordan MI. 2003.
Latent Dirichlet Allocation.
\textit{J. Mach. Learn. Res.}. 3, 993-1022

\bibitem{Blei2009}
Blei DM, Lafferty J. 2009.
Topic Models.
\textit{In A. Srivastava and M. Sahami, editors, Text Mining: Classification, Clustering, and Applications . Chapman $\&$ Hall/CRC Data Mining and Knowledge Discovery Series}.

\bibitem{Shen-Orr2010}
 Shen-Orr SS,  Tibshirani R,   Khatri, P,  Bodian DL,  Staedtler F,  Perry NM,  Hastie, T,   Sarwal MM,  Davis MM,  Butte AJ. 2010.
 Cell type specific gene expression   differences   in   complex   tissues.   
 \textit{Nature   Methods}.  7(4),   287-289  
 
\bibitem{Qiao2012}
Qiao W,  Quon G, Csaszar E, Yu  M,  Morris Q,  Zandstra  PW. 2012.   
PERT: A   Method   for   Expression   Deconvolution   of   Human   Blood   Samples   from   Varied
Microenvironmental   and   Developmental   Conditions.   
\textit{PLoS   Comput   Biol}.  8(12)
 
 \bibitem{Repsilber2010}
 Repsilber D,  Kern S,  Telaar A,  Walzl G,  Black GF,   Selbig   J,   Parida  SK,  Kaufmann SH,  Jacobsen M. 2010.
 Biomarker discovery in heterogeneous tissue samples -taking the in-silico deconfounding approach. 
 \textit{BMC bioinformatics}.11(1), 27+
 
 \bibitem{Schwartz2010}
 Schwartz R, Shackney SE. 2010.
 Applying unmixing to gene expression data for tumor phylogeny inference. 
 \textit{BMC bioinformatics}.  11(1), 42+
 
 \bibitem{Lindsay2013}
 Lindsay J,  Mandoiu I, Nelson C. 2013.
 Gene Expression Deconvolution using Single-cells
 \url{http://dna.engr.uconn.edu/bibtexmngr/upload/Lal.13.pdf}.
 
 \bibitem{Wang2015}
 Wang N, Gong T, Clarke R, Chen L, Shih IeM, Zhang Z, Levine DA, Xuan J, Wang Y.  2015.
 UNDO: a Bioconductor R package for unsupervised deconvolution of mixed gene expressions in tumor samples.
\textit{Bioinformatics}. 31(1): 137-9

\bibitem{Ahn2013}
Ahn J, Yuan Y, Parmigiani G, Suraokar MB, Diao L, Wistuba II, Wang W.
DeMix: deconvolution for mixed cancer transcriptomes using raw measured data. 2013.
\textit{Bioinformatics}. 29(15): 1865-71

\bibitem{Quon2013}
Quon G, Haider S, Deshwar AG, Cui A, Boutros PC, Morris Q.
Computational purification of individual tumor gene expression profiles leads to significant improvements in prognostic prediction.
\textit{Genome Med.}. 5(3): 29


 \bibitem{Hu2016}
 Hu JG,  Shi LL,  Chen YJ,  Xie XM,  Zhang N,  Zhu AY,  Zheng JS,  Feng YF,  Zhang C,  Xi J,  Lu HZ. 2016.
 Differential effects of myelin basic protein-activated Th1 and Th2 cells on the local immune microenvironment of injured spinal cord.
 \textit{Experimental Neurology}. 277, 190-201
 
 \bibitem{duVerle2016}
 duVerle D, Tsuda K. 2016.
 cellTree: Inference and visualisation of Single-Cell RNA-seq data as a hierarchical tree structure. 
 \textit{R package version 1.1.0}, \url{http://tsudalab.org}.
 
 \bibitem{Dey2016}
 Dey K, Hsiao J, Stephens M. 2016.
{\tt CountClust} : Clustering and Visualizing RNA-Seq Expression Data using Grade of Membership Models.
 \textit{R package version 0.99.3}, \url{https://www.bioconductor.org/packages/3.3/bioc/html/CountClust.html}
 
 
 \bibitem{Renard2013}
 Renard M,  Callewaert B,  Baetens M,  Campens L,  MacDermot K et al. 2013.
 Novel MYH11 and ACTA2 mutations reveal a role for enhanced TGF$\beta$ signaling in FTAAD
 \textit{Int J Cardiol}. 165(2), 314-321.
 
 \bibitem{Gong2006}
 Gong B,  Cao Z,  Zheng P,  Vitolo OV,  Liu S,  Staniszewski A,  Moolman D,  Zhang H,  Shelanski M,  Arancio O. 2006.
 Ubiquitin Hydrolase Uch-L1 Rescues $\beta$-Amyloid-Induced Decreases in Synaptic Function and Contextual Memory
 \textit{Cell}. 126(4), 775-788
 
 \bibitem{Hoffman2010}
 Hoffman MD,  Blei DM,  Bach F. 2010. 
 Online learning for latent Dirichlet allocation.
 \textit{Neural Information Processing Systems}.
 
 \bibitem{Kamburov2013}
 Kamburov A,  et al. 2013.
 The ConsensusPathDB interaction database: 2013 update. 
 \textit{Nucleic Acids Res}.
 
 \bibitem{Pentchev2010}
 Pentchev K,  et al. 2010.
 Evidence mining and novelty assessment of protein-protein interactions with the ConsensusPathDB plugin for Cytoscape. 
 \textit{Bioinformatics}.
 
 \bibitem{Guo2010}
Guo G, Huss M, Tong GQ, Wang C, Sun LL, Clarke ND, Robson P. 
Resolution of cell fate decisions revealed by single-cell gene expression analysis from zygote to blastocyst. \textit{Developmental Cell}. 18(4), 675-685 
    
\bibitem{Hou2007}
Hou J, Charters AM, Lee SC, Zhao Y, Wu, MK, Jones SJM, Marra, MA, Hoodless PA. 
A systematic screen for genes expressed in definitive endoderm by Serial Analysis of Gene Expression (SAGE). 
\textit{BMC Developmental Biology}. 7(92), 1-13 


\bibitem{Yoon2009}
Yoon S, Kim E, Kim YS, Lee H, Kim K, Bae J, Lee K. 
Role of Bcl2-like 10 (\textit{Bcl2l10}) in regulating mouse oocyte maturation.
\textit{Biology of Reproduction}. 81(3), 497-506. 

\bibitem{Evsikov2009}
Evsikov AV, De Evsikova C.  
Gene expression during the oocyte-to-embryo transition in mammals. 
\textit{Molecular Reproduction and Development}. 76, 805-818. 

\bibitem{Rossant1995}
Rossant J. 
Development of the extraembryonic lineages.
\textit{Seminars in Developmental Biology}. 6(4), 237-247.
 
\bibitem{Falco2007}
Falco G, Lee S, Stanghellini I, Bassey UC, Hamatani T, Ko MSH.
Zscan4: a novel gene expressed exclusively in late 2-cell embryos and embryonic stem cells.
\textit{Developmental biology}. 307(2), 539-550.

\bibitem{Schaniel2009}
Schaniel C, Ang YS, Ratnakumar K, Cormier C, James T, Bernstein E, Lemischka IR, Paddison PJ.
Smarcc1/Baf155 couples self-renewal gene repression with changes in chromatic structure in mouse embroynic stem cells.
\textit{Stem cells}. 27(12), 2979-91.

\bibitem{Putkey2002}
Putkey FR, Cramer T, Morphew MK, Silk AD, Johnson RS, Mclntosh JR, Cleveland.
Unstable Kinetochore-Microtubule capture and chromosomal instability following deletion of CENP-E.
\textit{Developmental cells}. 3(3), 351-365.

\end{thebibliography}

