\section{Methods Overview}

We assume that the RNA-seq data on $N$ samples has been summarized by a table of counts $C_{N \times G} = (c_{ng})$, 
where $c_{ng}$ is the number of reads from sample $n$ mapped to gene $g$ (or other unit, such as transcript or exon) \cite{Oshlack2010}.  
The GoM model is a generalization of a cluster model, which allows that each sample has some proportion (``grade") of membership,
in each cluster.  For RNA-seq data this corresponds to 
assuming that each sample $n$ has some proportion of its reads, $q_{nk}$ coming from cluster $k$. 
In addition, each cluster $k$ is characterized by a probability vector, $\theta_{k\cdot}$, whose $g$th element represents
the relative expression of gene $g$ in cluster $k$. 
The GoM model is then
\begin{equation} \label{eqn:mult}
\left ( c_{n1}, c_{n2}, \cdots, c_{nG} \right) \sim \text{Multinomial} \left(c_{n+}, p_{n1}, p_{n2}, \cdots, p_{nG} \right),
\end{equation}
where $c_{n+} = \sum_{g=1}^{G} c_{ng}$ and we assume
\begin{equation} \label{eqn:gom}
p_{ng} := \sum_{k=1}^{K} q_{nk}\theta_{kg}.    
\end{equation}
The number of clusters $K$ is set by the analyst, and it can be helpful to explore multiple
values of $K$ (see Discussion).

To fit this model to RNA-seq data, we exploit the fact that exactly the same GoM model is commonly used for document clustering \cite{Blei2003}.
This is because, just as RNA-seq samples can be summarized by counts of reads mapping to each possible gene in the genome, 
document data can be summarized by counts of each possible word in a dictionary.
Recognizing this allows existing methods and software for document clustering to be applied directly to RNA-seq data. 
%Specifically,  the  Latent Dirichlet Allocation model from \cite{Blei2003}, which is widely used to cluster documents based on their word counts, 
%is based on a multinomial model that applies naturally and immediately to RNA-seq data. 
Here we use the R package {\tt maptpx} \cite{Taddy2012} to fit the GoM model.

Fitting the GoM model results in estimated membership proportions $q$ for each sample, and estimated expression values $\theta$ for each cluster.
We visualize the membership proportions for each sample using a ``Structure plot" \cite{Rosenberg2002}, 
which is named for its widespread use in visualizing the
results of the {\it Structure} software \cite{Pritchard2000} in population genetics.
The Structure plot represents the estimated membership proportions of each sample 
as a stacked barchart, with bars of different colors representing  different clusters. Consequently, samples that have similar membership proportions have
similar amounts of each color. See Fig~\ref{fig:fig1} for example.
%In addition, to help biologically interpret each cluster, we developed a method to identify which genes are most distinctively differentially expressed in each cluster (see Methods).

To help biologically interpret the clusters inferred by the GoM model we also implemented methods to identify, for each cluster, which genes
are most distinctively differentially expressed in that cluster (see Methods). Functions for fitting the GoM model, plotting the
structure plots, and identifying the distinctive (``driving") genes in each cluster, are included in our R package {\tt CountClust} \cite{Dey2016} available through Bioconductor \cite{Gentleman2003}.  

\section{Results}

%An outline for results  (under consideration)
%
%\begin{itemize}
%
%\item Form two separate subsections, one for the GTEx Version 4 data and the other for the single cell Zeisel data.
%
%\item For GTEx data, give a figure comprising of 4 Structure plots for different $K$s, may be $2,5,10,15$. Fix the thinning parameter $p_{thin}$ to say $0.0001$.  Also record the log likelihoods (Bayes factors) for each of the 4 models, as reported by \textbf{maptpx}. 
%
%\item Have one figure showing the robustness of the clustering method on the thinning parameter $p_{thin}$. Fix $k=10$ and vary $p_{thin}$ to be $0.0001$, $0.001$ and $0.01$. 
%
%\item One t-SNE plot for GTEx samples (with and without admixture in the same plot). Should this be in results or in discussions? Also the t-SNE probably would require an electronic supplemental file as I would need the \textbf{qtlcharts} highlighting for those plots. 
%
%\item The GTEX brain samples Structure plot  for $K=4$ that shows the neuron cell types in brain cerebellum and cerebellar hemisphere. That is to show that the clusters are driven by cell types.
%
%\item Gene annotations for the GTEx significant genes (for brain) and also for the general set up (to decide on which $K$ to fix). Use Bayes Factor?
%
%\item The Structure plot for Zeisel single cell data. again Multiple $K=2,5,7,10$. 
%
%\item Gene annotations for the Zeisel single cell data. Need to choose the optimal $K$. Use Bayes Factor?
%
%\item t-SNE plot of the admixture proportions??..Is that required? Depends on how we present t-SNE. If this goes to discussion, we will avoid it here
%
%\end{itemize}


\subsection{Clustering human tissue samples using bulk RNA-seq}

We begin by illustrating the GoM model on bulk RNA expression measurements from the GTEx project (V6 dbGaP accession phs000424.v6.p1, release date: Oct 19, 2015, \url{http://www.gtexportal.org/home/}).  These data consist of per-gene read counts from RNA-seq performed on $8,555$ samples collected from $450$ human donors across $51$ tissues, lymphoblastoid cell lines, and transformed fibroblast cell-lines. We analyzed $16,069$ genes that satisfied filters (e.g.~exceeding certain minimum expression levels) that were used during eQTL analyses by the GTEx project (gene list available in \url{http://stephenslab.github.io/count-clustering/project/utilities/gene_names_all_gtex.txt}). 

We fit the GoM model to these data, with number of clusters $K=5,10,15,20$. For each $K$ we ran the fitting algorithm three times and 
kept the result with the highest log-likelihood. Fig~\ref{fig:fig1}(a) shows the Structure plot for $K=20$, with results for other $K$ in \nameref{figS0}.
(See also \url{http://stephenslab.github.io/count-clustering/project/src/tissues_tSNE_2.html} for 
an alternative visualization using a 2-dimensional projection with t-SNE \cite{Maaten2008, Maaten2014}.)

In all cases results reflect the known division of samples into tissues: that is, samples from the same tissue tend to have similar cluster membership proportions.
As might be expected, increasing $K$ highlights finer structure in the data, with tissues that cluster together with smaller $K$ being subdivided into distinct subgroups
for larger $K$. For brevity we focus on results for $K=20$ (Fig~\ref{fig:fig1}(a)). Here some tissues are represented by essentially a single cluster (e.g.~Pancreas, Liver), whereas other tissues are represented as a mixture of multiple clusters (e.g.~Thyroid, Spleen). Furthermore, the results highlight biological similarity among some tissues by assigning similar membership proportions to samples from those tissues.  For example, samples from several different parts of the brain often have similar memberships, as do the arteries (aorta, tibial and coronary) and skin samples (sun-exposed and un-exposed). 

To help biologically interpret results we implemented methods to identify the genes and genetic processes that characterize each cluster (see Methods).
%Although we allow that these distinguishing genes may be either over-expressed or under-expressed in one cluster compared to other clusters,  in practice the vast majority of genes identified here are due to over-expression. We then examined the function of these genes, both individually, and collectively -- using ``enrichment analysis" to identify biological processes that are common to the genes distinguishing each cluster.   
Table~\ref{tab:tab1} summarizes results for the GTEx results in Fig~\ref{fig:fig1}a (see also \nameref{supptab1}). Reassuringly, many results align with known biology. For example,  the purple cluster (cluster 18), which distinguishes Pancreas from other tissues, is enriched for
genes responsible for digestion and proteolysis, (e.g. \textit{PRSS1}, \textit{CPA1}, \textit{PNLIP}). Similarly the yellow cluster (cluster 12), which primarily distinguishes Cell EBV Lymphocytes from other tissues, is enriched with genes responsible for immune responses (e.g. \textit{IGHM}, \textit{IGHG1}) and the pink cluster (cluster 19) which mainly shows up in Whole Blood, is enriched with genes related hemoglobin complex and oxygen transport (e.g. \textit{HBB}, \textit{HBA1}, \textit{HBA2}). Further, Keratin-related genes characterize the skin cluster (cluster 6, light denim), Myosin-related genes characterize the muscle skeletal cluster (cluster 7, orange), etc. The royal purple cluster (cluster 1) has memberships in most tissues and the genes distinguishing the cluster seem to be responsible for  nucleus and nucleoplasm related functionality. In cases where a cluster occurs in multiple tissues these biological annotations may be particularly helpful for understanding what is driving this co-membership. For example, the top genes in the red cluster (cluster 3), which is common to Breast Mammary tissue, Adipose Subcutaneous and Adipose Visceral, are related to adipocytes and/or fatty acid synthesis; and the top genes in the salmon cluster (cluster 4), which is common to the Gastroesophageal Junction, Esophagus Muscularis and Colon Sigmoid, are related to smooth muscle.


%A field of very active interest in recent times is to estimate the proportion of different cell types in different tissues. Marker based approaches are usually adopted to validate for different cell types and get a sense of the abundance of different cell types in the tissue samples \cite{Grun2015} \cite{Palmer2005}. The admixture model is a marker free method to obtain clusters driven by cell types. 

Although global analysis of all tissues is useful for highlighting major structure in the data, it may miss finer-scale structure within tissues or among similar tissues. For example, here the global analysis allocated only three clusters to all brain tissues (clusters 1,2 and 9 in Fig~\ref{fig:fig1}(a)), and we suspected that additional substructure might be uncovered by analyzing the brain samples separately with larger $K$.  Fig~\ref{fig:fig1}(b) shows the Structure plot for $K=6$ on only the Brain samples. The results highlight much finer-scale structure compared with the global analysis. Brain Cerebellum and Cerebellar hemisphere are essentially assigned to a separate cluster (lime green), which is enriched with genes related to cell periphery and communication (e.g. \textit{PKD1}, \textit{CBLN3}) as well as genes expressed largely in neuronal cells and playing a role in neuron differentiation (e.g. \textit{CHGB}). The spinal cord samples also show consistently strong membership in a single cluster (yellow-orange), the top defining gene for the cluster being \textit{MBP} which is involved in myelination of nerves in the nervous system\cite{Hu2016}.  Another driving gene, \textit{GFAP}, participates in system development by acting as a marker to distinguish astrocytes during development \cite{Baba1997}.


%This cluster seems likely to reflect the expected high concentration of Cerebellar granule cells in cerebellar samples.
%[POssibly GRANULE CELLS?]
% (Genetic mutations in both MYH11 and ACTA2 can cause Thoracic aortic aneurysms/dissection \cite{Renard2013}.) 

The remaining samples all show membership in multiple clusters. Samples from the putamen, caudate and nucleus accumbens show similar profiles, and are distinguished
by strong membership in a cluster (cluster 4, bright red) whose top driving gene is \textit{PPP1R1B}, a target for dopamine.
And cortex samples are distinguished from others by stronger membership in a cluster (cluster 2, turquoise in Fig~\ref{fig:fig1}(b)) whose distinctive genes include \textit{ENC1}, which interacts with actin and contributes to the organisation of the cytoskeleton during the specification of neural fate \cite{Hernandez1997}.



% It seems the red cluster is mainly prominent in what are called sub-cortical regions of the brain (which are immediately below the cortex).  In that sense the four clusters, although not fully representative, do seem to be driven by the cerebellum, cortex, sub-cortex and spinal cord. These are anatomically meaningful and also functionally.  Functionally, sub-cortical regions of the brain are associated with lower order thinking tasks and these are regions that are present in reptiles and birds as well. Cortex is mainly present in mammals and is used for higher order thinking tasks. Cerebellum main function is in motion learning. Also amygdala and hippocampus are anatomically very close to each other and constitute the limbic region of the brain \url{https://s3.amazonaws.com/classconnection/118/flashcards/3700118/png/screen_shot_2012-09-17_at_75637_am1347883157040-1517F5218E616F44DE2.png}. Even substantia nigra is pretty close to both of them. Also cerebellar granule cells do constitute bulk of the cerebellum and are the principal granule cells in human body (WIKI). Given that the genes we obtained as driving genes for the clusters seem to be associated with cerebellar granule cells, seems it is meaningful to call the green cluster to be driven by the granule cells. Myelin basic protein has been found to have higher expression in spinal cord cluster when compared to other clusters. Myelin increases nervous transmission and nervous transmission is very swift through the spinal cord. This could explain why MBP is one of the top distinguishing genes for the spinal cord cluster. 




%Recent stereological approaches have shown that rat cerebellum contains $> 80 \%$ neurons (Herculano-Houzel and Lent 2005) \cite{Houzel2005}, much higher than other parts of the brain. 

\subsection{Quantitative comparison with hierarchical clustering}

%The GoM model is complimentary to, rather than only competing with, distance-based hierarchical clustering methods.
%Nonetheless, 
We used the GTEx data to test whether the GoM model is more accurate in detecting substructure than distance-based clustering methods. 
Specifically, for each pair of tissues in the GTEx data we assessed whether or not each clustering method
correctly partitioned samples into the two tissue groups (see Methods). The GoM model was substantially more accurate in this test, succeeding
in $81 \%$ of comparisons, compared with $29 \%$ for the distance-based method (Fig~\ref{fig:fig2}). This presumably reflects the general tendency for model-based
approaches to be more efficient that distance-based approaches, provided that the model is sufficiently accurate.


\subsection{Clustering of single-cell RNA-seq data}

Recently RNA-sequencing has become viable for single cells \cite{Tang2009}, and this technology has the promise to revolutionize understanding of intra-cellular variation in expression, and regulation more generally \cite{Trapnell2015}. Although it is traditional to describe and categorize cells in terms of distinct cell-types, 
the actual architecture of cell heterogeneity may be more complex, and in some cases perhaps better captured by the more ``continuous"  GoM model. 
In this section we illustrate the potential for the GoM model to be applied to single cell data.

To be applicable to single-cell RNA-seq data, methods must be able to deal with lower sequencing depth than in bulk RNA experiments:
 single-cell RNA-seq data typically involve substantially lower effective sequencing depth compared with bulk experiments, due to the relatively small 
 number of molecules available to sequence in a single cell. Therefore, as a first step towards demonstrating its potential for single cell analysis,
  we checked robustness of the GoM model to sequencing depth. Specifically,
we repeated the analyses above after thinning the GTEx data by a factor of $100$ and $10,000$ to mimic the lower sequencing depth of a typical single cell experiment.
 %per-sample depth in the 
%single-cell data from \cite{Jaitin2014} considered below. (See  \textbf{Figure \ref{fig:figS1}} and \textbf{Figure \ref{fig:figS2}} for results with other values of $p_{thin}$.) 
For the thinned GTEx data the Structure plot for $K=20$ preserves most of the major features of the original analysis on unthinned data (\nameref{figS1}). For the accuracy comparisons with distance-based methods, both methods suffer reduced accuracy in thinned data, but the GoM model remains superior (\nameref{figS2}). For example, when thinning by a factor of $10,000$, the success rate in separating pairs of tissues is $0.32$ for the GoM model vs $0.10$ for hierarchical clustering.

% (for $p_{thin}=0.01$, the success proportion in separating two tissues is $0.11$  for hierarchical clustering and $0.36$ for GoM model; for $p_{thin}=0.001$, the success proportion in separating tissues is  $0.10$ for hierarchical clustering and $0.32$ for GoM model).  

 %Then we generated a set of $50$ samples randomly drawn from the pooled set of samples coming from these two tissues and then observed whether the hierarchical and the admixture were separating out samples coming from the two different tissues.

Having established its robustness to sequencing depth,
we now illustrate the GoM model on two single cell RNA-seq datasets, from Jaitin \textit{et al} \cite{Jaitin2014} and Deng \textit{et al} \cite{Deng2014}.  

\subsubsection{Jaitin \text{et al}, 2014}

Jaitin \textit{et al} sequenced over $4,000$ single cells from mouse spleen. 
%Following the original authors protocol, we filtered out 16 genes that they found to show significant batch-specific expression. 
Here we analyze $1,041$ of these cells that were categorized as $CD11c+$ in the \textit{sorting markers} column of their data (\url{http://compgenomics.weizmann.ac.il/tanay/?page_id=519}), and which had total number of reads mapping to non-ERCC genes greater than $600$. We believe these cells correspond roughly to the $1,040$ cells in their Figure S7.   Our hope was that applying our method to these data would identify, and perhaps refine, the cluster structure evident in 
\cite{Jaitin2014} (their Figures 2A and 2B). However, our method yielded rather different results (Fig~\ref{fig:fig3}), where most cells were assigned to have membership
in several clusters. Further, the cluster membership vectors showed systematic differences among amplification batches (which in these data is also strongly correlated with sequencing batch). For example, cells in batch 1 are characterized by strong membership in the orange cluster (cluster 5) while those in batch 4 are characterized
by strong membership in both the blue and yellow clusters (2 and 6). Some adjacent batches show similar patterns - for example batches 28 and 29 have a similar visual ``palette", as do batches 32-45. And, more generally, these later batches are collectively more similar to one another than they are to the earlier batches (0-4).

The fact that batch effects are detectable in these data is not particularly surprising: there is a growing recognition of the importance of batch effects in high-throughput data generally \cite{Leek2010} and in single cell data specifically \cite{Hicks2015}. And indeed, both clustering methods and the GoM model can be viewed
as dimension reduction methods, and such methods can be helpful in controlling for batch effects \cite{Leek2007, Stegle2012}. However, why these batch effects are not evident in Figures 2A and 2B of \cite{Jaitin2014} is unclear. 


%identify distinct clusters, corresponding to the clusters of B cells, NK cells, pDCs and monocytes
%focusing initially on a heterogeneous mix enriched for expression of the CD11c surface marker. 
% The aim of their study was to separate out the B cells, NK cells, pDCs and monocytes.   \textbf{Fig \ref{fig:fig3}} (\textit{top panel}) presents the Structure plot  for $K=7$ for the Jaitin \textit{et al} data with the samples arranged by their amplification batch (which was a refinement of the sequencing batch). This highlights the need for caution regarding interpreting Admixture results or any clustering results, as there is a possibility of technical effects driving the clusters instead of true biological effects. There has been a growing concern among biostatisticians today about how to deal with batch effects \cite{Leek2010} \cite{Hicks2015}. \\[1 pt]

% Deng \textit{et al} collected expression data from individual cells from zygote to blastocyst stages of mouse preimplantation development \cite{Deng2014}. Deng \textit{et al}'s analysis focussed particularly on allele-specific expression from the two contributing mouse strains (CAST/EiJ and C57BL/6J). Here we analyze the counts of the two alleles combined. Visual inspection of the Principal Components Analysis in \cite{Deng2014} suggested 6-7 clusters, so we fit the cluster model with $K=6$. 
% The results (Figure \ref{fig:fig4}) clearly highlight the structure in the different development stages starting from zygote, through early/mid/late 2 cells, 4 cells, 8 cells, 16 cells, and early/mid blastocyst to finally late blastocyst. Specifically, cells that are from the same stage show similar cluster membership proportions. Further, many of the clusters show notable trends through the stages. For example, 
% membership in the green cluster is non-existent in early stages, starts in the 4-cell stage, becomes more prominent in the 8-16 cell stages, drops substantially in the early and mid-blastocyte stages, and is essentially absent in the late blastocytes. More generally, cluster memberships for cells from adjacent stages tend to be more similar to one another than those for cells from distant stages. 

% Examining the clustering results by embryo highlights apparent embryo-level effects in the early stages (Figure \ref{fig:fig4}): that is, cells from the same embryo sometimes showed distinctive differences from other embryos. For example, the two cells from one of the 2-cell embryos (check) shows much stronger membership in the magenta cluster than other 2-cell embryos, and four cells from one of the 4-cell embryos (embryo 4) shows consistently more yellow membership than the other 4-cell embryos. 

% Finally, the results indicate a few samples that appear to be outliers - for example, a cell from a 16-cell embryo that looks like a very early stage cell (zygote or early 2-cell), and a cell from an 8-stage embryo that looks rather different from any of the others.

\subsubsection{Deng \text{et al}, 2014}

Deng \textit{et al} collected single-cell expression data of mouse preimplantation embryos from the zygote to blastocyst stage \cite{Deng2014},
with cells from four different embryos sequenced at each stage. The original analysis \cite{Deng2014} focusses on trends of allele-specific expression in early embryo development. Here we use the GoM model to assess the primary structure in these data without regard to allele-specific effects 
(i.e.~combining counts of the two alleles). Visual inspection of the Principal Components Analysis in \cite{Deng2014} suggested perhaps 6-7 clusters, 
and we focus here on results with $K=6$. 

The results from the GoM model (Fig~\ref{fig:fig4}) 
clearly highlight changes in expression profiles that occur through early embryonic development stages, and enrichment analysis of
the driving genes in each cluster (Table~\ref{tab:tab3},  \nameref{supptab3}) indicate that many of these expression changes reflect important
biological processes during embryonic preimplantation development. 

In more detail: Initially, at the zygote and early 2-cell stages, the embryos are represented by a single cluster (blue in Fig~\ref{fig:fig4}) that
 is enriched with genes responsible for germ cell development (e.g., \textit{Bcl2l10} \cite{Yoon2009}, \textit{Spin1} \cite{Evsikov2009}). Moving through
 subsequent stages the grades of membership evolve to a mixture of blue and magenta clusters (mid 2-cell), a mixture of magenta and yellow clusters 
(late 2-cell) and a mixture of yellow and green (4-cell stage). The green cluster then becomes more prominent in the 8-cell and 16-cell stages, before dropping  substantially in the early and mid-blastocyst stages. That is, we see a progression in the importance of different clusters
through these stages, from the blue cluster, moving through magenta and yellow to green. By examining the genes distinguishing each cluster
we see that this progression reflects the changing relative importance of several fundamental biological processes.
The magenta cluster is driven by genes responsible for the beginning of transcription of zygotic genes (e.g., \textit{Zscan4c-f} show up in the list of top $100$ driving genes : see \url{https://stephenslab.github.io/count-clustering/project/src/deng_cluster_annotations.html}), which takes place in the late 2-cell stage of early mouse embryonic development \cite{Falco2007}. The yellow cluster is enriched for genes responsible for heterochromation \textit{Smarcc1} \cite{Schaniel2009} and chromosome stability \textit{Cenpe} \cite{{Putkey2002}} (see \nameref{supptab3}) . And the green cluster is enriched for cytoskeletal genes (e.g., \textit{Fbxo15}) and cytoplasm genes (e.g., \textit{Tceb1}, \textit{Hsp90ab1}), all of which are essential for compaction at the 8-cell stage and morula formation at the 16-cell stage. 

Finally, during the blastocyst stages two new clusters (purple and orange in Fig~\ref{fig:fig4}) dominate.
The orange cluster is enriched with genes involved in the formation of outer trophoblast cells (e.g., \textit{Tspan8}, \textit{Krt8}, \textit{Id2} \cite{Guo2010}), while the purple cluster is enriched with genes responsible for the formation of inner cell mass (e.g., \textit{Pdgfra}, \textit{Pyy} \cite{Hou2007}). 
Thus these two clusters are consistent with the two cell lineages, the trophectoderm
and the primitive endoderm, that make up the majority of
the cells of the blastocyst \cite{Rossant1995}. Interestingly, however, the cells do not appear to fall into two distinct and clearly-separated populations
-- at least, not in terms of their expression patterns -- but rather show a continuous
range of memberships in these two clusters, even in the late blastocyst stage.

In addition to these trends across development stages, the GoM results also highlight some embryo-level effects in the early stages (Fig~\ref{fig:fig4}). 
Specifically, cells from the same embryo sometimes show greater similarity than cells from different embryos.
%Four embryos were included and collected from 4 to 6 mice at each stage. 
%For example, cells from the four embryos collected at the 16-cell stage are represented by a mixture of blue, green, yellow, and purple clusters. 
For example, while all cells from the 16-cell stage have high memberships in the green cluster, cells from two of the embryos at this stage have memberships in both the purple and yellow clusters, while the other two embryos have memberships only in the yellow cluster. 

Finally, we note that, like clustering methods, the GoM model can be helpful in exploratory data analysis and quality control. Indeed, 
the GoM results highlight a few single cells as outliers. For example, a cell from a 16-cell embryo is represented by the blue cluster - a cluster that represents cells at the zygote and early 2-cell stage. Also, a cell from an 8-stage embryo has strong membership in the purple cluster - a cluster that represents cells from the blastocyst stage. 
It would seem prudent to consider excluding these cells from subsequent analyses of these data.
%and cleavage division (e.g., \textit{Tfgb2}, \textit{Pttg1}), 

%Notably, for both these single-cell data sets, most cells are assigned to a combination of more than one cluster, rather than a single cluster (the exception being the very early-stage cells in data from \cite{Deng2014}). This highlights the potential utility for GoM models to capture structure in single cell data that might be missed by simpler cluster-based approaches.

%Code for reproducing the results reported here is available at \url{http://stephenslab.github.io/count-clustering/}.




%, 
%We find that admixture model is more successful in separating out different tissues in general, compared to the hierarchical clustering technique. The admixture model is essentially a count based modeling approach and seems to handle low counts and zero counts much better than the hierarchical method which is a more general approach to clustering. Since the RNA-seq data and in particular scRNA-seq data have lots of low counts and zero counts, the admixture model seems to be more suited for such data compared to hierarchical clustering method. 

%Currently there is a lot of interest in single cell sequencing as it is more informative about individual cell expression profiles compared to the RNA-seq on tissue samples. We were curious to see how stable the Admixture results are if the GTEx RNA-seq data is viewed at the scale of a single cell data. We achieve the latter by thinning the GTEx data under thinning parameter $p_{thin}=0.0001$ which is the order of scale obtained by dividing the total library size of the Jaitin \textit{et al} \cite{Jaitin2014} with respect to the library size of the GTEx V4 read counts data. We fitted the admixture model for $K=12$ on the thinned data and the Structure plot for the fitted model is presented in \textbf{Fig \ref{fig:fig4}}. It seems that most of the features observed in \textbf{Fig \ref{fig:fig1}} seem to be retained, for instance- the Brain samples clustering together, Whole blood and Testis forming separate clusters, Muscle skeletal and Heart tissue samples showing very similar patterns etc. However, thinning indeed shrinks the small differences across tissues and makes it more difficult to distinguish between tissues, as evident from the comparative study of hierarchical and admixture models, analogous to \textbf{Fig \ref{fig:fig3}}, for thinned data with thinning parameters $p_{thin}=0.001$ and $p_{thin}=0.0001$ in  \textbf{Fig \ref{fig:figS2}}. One can see that with thinning, the performance of admixture model in separating the tissues deteriorates but encouragingly, it seems that admixture does outperform the hierarchical clustering even under thinned data. \\[3pt]

 








